README.md

```diff
@@ -90,7 +90,7 @@ Benchmark-operator includes a series of end 2 end tests that can be triggered in
 Optional locations for workload images can now be added easily without the need to rebuild the operator.
 To do so in the workload args section of the CR add image: [location]
 
-NOTE: This is not a required arguement. If omitted it will default to the currently verified workload image.
```

Comment:
I believe "omitted" is correct in both places, as it's in the context of "if the argument is omitted".

Reply:
You are correct! I accidentally added Committed thinking it was grammatically correct!